recipes/glfw/all/conanfile.py

```diff
@@ -23,49 +27,84 @@ class GlfwConan(ConanFile):
         "shared": [True, False],
         "fPIC": [True, False],
         "vulkan_static": [True, False],
+        "with_x11": [True, False],
+        "with_wayland": [True, False],
     }
     default_options = {
         "shared": False,
         "fPIC": True,
         "vulkan_static": False,
+        "with_x11": True,
+        "with_wayland": True,
```

Comment:
Is it possible to use both Wayland and X11 at same time?

Reply:
Yes, it is and in fact up to version `3.3.8`, `X11` support can't be disabled. However, upstream changes now allow selectively choosing one, the other, or both as desired. I've done some work to prepare for the next version ahead of time to make it easier to update the recipe in the future.